### PR TITLE
compress kitchen verify log

### PR DESCRIPTION
**Pull request by bryceml**

### CODE REVIEW

**Comment by bryceml** [comment]
So currently the verify file is only 5M where as on the pytest prs they are 427M
If we want space savings currently, we would want to compress the runtests log as well.

File: vars/runTestSuite.groovy [hunk 1]
```diff
@@ -268,8 +268,17 @@ def call(Map options) {
...
+                        sh """
+                        # Do not error if there are no files to compress
+                        xz ".kitchen/logs/*-verify.log" || true
+                        xz "artifacts/logs/*" || true
+                        """
+                    }
+
                     archiveArtifacts(
-                        artifacts: "artifacts/*,artifacts/**/*,.kitchen/logs/*-create.log,.kitchen/logs/*-converge.log,.kitchen/logs/*-verify.log,.kitchen/logs/*-download.log,artifacts/xml-unittests-output/*.xml",
+                        artifacts: "artifacts/*,artifacts/**/*,.kitchen/logs/*-create.log,.kitchen/logs/*-converge.log,.kitchen/logs/*-verify.log,.kitchen/logs/*-verify.log.xz,.kitchen/logs/*-download.log,artifacts/xml-unittests-output/*.xml",
```
Comment (s0undt3ch) [inline vars/runTestSuite.groovy]: `.kitchen/logs/*-verify.log*,`

**Comment by s0undt3ch** [comment]
We can compress all .kitchen/logs, and then conditionally compress runtests logs.
The verify log will have what we see on Jenkins console, plus some other minor stuff.

**Comment by bryceml** [comment]
the other kitchen logs besides verify are at most a few k, so I think I'll leave them uncompressed.

**Comment by bryceml** [comment]
https://jenkinsci.saltstack.com/job/pr-archlts-py2/job/master/166/console is running with this change using a replay
https://jenkinsci.saltstack.com/job/pr-docs/job/master/179/console
https://jenkinsci.saltstack.com/job/pr-lint/job/master/170/console as well

**Comment by bryceml** [comment]
looks like the if statement didn't work.

**Comment by bryceml** [comment]
https://jenkinsci.saltstack.com/job/pr-archlts-py2/job/master/179/ works now.

**Comment by bryceml** [comment]
We compress by tailing and greping the runtests log to see if it had an exit code of 0 and if it does, we compress it.

**Pull request merged**